From d7ebc96446b2fd5f835ab853b9a95ac581480e67 Mon Sep 17 00:00:00 2001
From: Steve Wainstead <wainstead@gmail.com>
Date: Tue, 26 Jul 2011 05:55:10 +0000
Subject: [PATCH 180/223] Bumped up the version number of Waverous; tweaked
 the output of one of the custom macros; removed an
 errant comment sign

---
 server/configure.ac |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/server/configure.ac b/server/configure.ac
index 5235fde..84ad7ee 100644
--- a/server/configure.ac
+++ b/server/configure.ac
@@ -16,7 +16,7 @@ dnl   Palo Alto, CA 94304
 dnl   Pavel@Xerox.Com
 dnl
 AC_PREREQ([2.61])
-AC_INIT([Waverous LambdaMOO], [beta1], [wainstead@gmail.com], [waverous-lambdamoo], [http://code.google.com/p/waverous/])
+AC_INIT([Waverous LambdaMOO], [beta2], [wainstead@gmail.com], [waverous-lambdamoo], [http://code.google.com/p/waverous/])
 AC_CONFIG_SRCDIR([functions.c])
 AC_CONFIG_HEADERS([config.h])
 
@@ -158,7 +158,7 @@ trfrom='[a-z]' trto='[A-Z]'
 changequote([,])dnl
 for var in $2
 do
-echo "checking whether $var is declared in $1"
+echo "moo_ndecl: checking whether $var is declared in $1"
 MOO_VAR_DECL_CHECK($1, $var, ,
 	AC_DEFINE(NDECL_`echo $var | tr "$trfrom" "$trto"`), $3)
 done
@@ -606,7 +606,7 @@ else
 fi
 
 
-dnl # let the user choose to enable ssl or not
+dnl Let the user choose to include openssl or not
 AC_ARG_WITH([openssl],
         [AS_HELP_STRING([--with-openssl], [include openssl (Secure Socket Layers) (experimental) @<:@default: no@:>@])],
         [openssl=${withval}],
-- 
1.7.9.5

